src/Microsoft.DocAsCode.Build.Engine/TemplateManager.cs

```diff
@@ -36,6 +37,20 @@ public TemplateProcessor GetTemplateProcessor(DocumentBuildContext context, int
             return new TemplateProcessor(new CompositeResourceCollectionWithOverridden(_templates.Select(s => _finder.Find(s)).Where(s => s != null)), context, maxParallelism);
         }
 
+        public string GetTemplatesHash()
+        {
+            var resources = _templates.Select(s => _finder.Find(s)).Where(s => s != null);
+            var builder = new StringBuilder();
+            foreach (var r in resources)
```

Comment:
why? templates order matters

---
In reply to: [75444228](https://github.com/dotnet/docfx/pull/621#discussion_r75444228) [](ancestors = 75444228)